Zapper-fi/studio
# feat(apps: morpho): Morpho Blue support

## Description



## Checklist

- [x] I have followed the [Contributing Guidelines](https://github.com/Zapper-fi/studio/blob/main/CONTRIBUTING.md)
- [x] (optional) As a contributor, my Twitter handle is: @TomReppelin

## How to test?


- Morpho Blue adapter. 

Test it by running 

`pnpm build && pnpm start`

and go there:

http://localhost:5001/apps/morpho/positions?network=ethereum&groupIds%5B%5D=morpho-blue

-----------------------------------
Important note: 
1. The wbIB01 token is not supported yet:
https://etherscan.io/address/0xcA2A7068e551d5C4482eb34880b194E4b945712F

Thus the market with the id:
0x495130878b7d2f1391e21589a8bcaef22cbc7e1fbbd6866127193b3cc239d8b1
(more info here: https://docs.morpho.org/addresses#morpho-blue-whitelisted-markets
is not supported)

2. The `collateral` computation can't be done easily onchain, thus the subgraph fetch is forced here

-----------------------------------
One should see something like this:
<img width="789" alt="image" src="https://github.com/morpho-labs/zapper-studio/assets/75627352/570ed12e-4d81-4c1b-8cee-5ffc1b5526d7">

Thanks!
